openpype/hosts/blender/api/render_lib.py

```diff
@@ -212,14 +213,13 @@ def imprint_render_settings(node, data):
 
 
 def prepare_rendering(asset_group):
-    name = asset_group.name
+    name = Path(asset_group.name)
 
-    filepath = bpy.data.filepath
+    filepath = Path(bpy.data.filepath)
     assert filepath, "Workfile not saved. Please save the file first."
 
-    file_path = os.path.dirname(filepath)
-    file_name = os.path.basename(filepath)
-    file_name, _ = os.path.splitext(file_name)
+    file_path = filepath.parent
```

Comment:
Having both `filepath` and file_path` variable in this scope feels a bit confusing. Should this be `filepath = filepath.parent` or `directory = filepath.parent`?

The parent isn't technically a filepath, it's the path to a directory/folder instead.

Reply:
Yep you're right! I'll change it :)